refinery/config/settings/base.py

```diff
@@ -600,3 +619,7 @@ def get_setting(name, settings=local_settings):
 
 # To avoid Port conflicts between LiveServerTestCases http://bit.ly/2pb64KN
 os.environ["DJANGO_LIVE_TEST_SERVER_ADDRESS"] = "localhost:10000-12000"
+
+DJANGO_DOCKER_ENGINE_BASE_URL = "visualizations"
```

Comment:
@hackdna Django url patterns always have the leading slash by default
See: https://docs.djangoproject.com/en/1.7/topics/http/urls/#example

Django yells at me if I do this (when specifying a leading slash prior):
```
url(
    r'^{}/'.format(settings.DJANGO_DOCKER_ENGINE_BASE_URL),
    include('django_docker_engine.urls')
)
```
Because the endpoint is then configured as `//visualizations/`